### PR TITLE
Add new UI (Original view on the left side, new Schedule view on the right side)

### PR DESCRIPTION
**Pull request by seanflyyy**
There are some things to follow up 
- Under MainWindow.java, there is a need to update the function for ScheduleListPanel to return only the persons in the day itself, rather than the current filteredPersonList. The method for getSchedule() has yet to be added.
- Under ScheduleCard.java, there is a need to update the classTime field, currently right now, on line 50, the class time is written as "1-2pm". This should be updated with person.getClassTime().value, which currently has yet to be implemented.

### CODE REVIEW

**Comment by seanflyyy** [comment]
Close #46

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -36,6 +36,8 @@ public class MainWindow extends UiPart<Stage> {
     private HelpWindow helpWindow;
     private StatisticsWindow statisticsWindow;
```
Comment (cadencjk) [inline src/main/java/seedu/address/ui/MainWindow.java:38]: Empty line between StatisticsWindow and ScheduleListPanel

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -45,6 +47,8 @@ public class MainWindow extends UiPart<Stage> {
     @FXML
     private StackPane personListPanelPlaceholder;
 
+    @FXML
+    private StackPane scheduleListPanelPlaceholder;
     @FXML
```
Comment (cadencjk) [inline src/main/java/seedu/address/ui/MainWindow.java:52]: Should there be an empty line between the 2 FXML objects?

File: src/main/java/seedu/address/ui/ScheduleCard.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    private HBox scheduleCardPane;
+    @FXML
+    private Label name;
+    @FXML
+    private Label id;
+    @FXML
+    private Label phone;
+    @FXML
+    private Label address;
+
```
Comment (cadencjk) [inline src/main/java/seedu/address/ui/ScheduleCard.java:36]: Empty line here.

File: src/main/java/seedu/address/ui/ScheduleCard.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Person person;
+
+    @FXML
+    private HBox scheduleCardPane;
```
Comment (cadencjk) [inline src/main/java/seedu/address/ui/ScheduleCard.java:27]: Just curious is there a typo is it called a pane. ie: I was thinking `scheduleCardPanel`

**Review by cadencjk — APPROVED** [review]
Other than minor cosmetic issues, the code LGTM!

File: src/main/java/seedu/address/ui/ScheduleCard.java (hunk 4 above)
Comment (seanflyyy) [reply src/main/java/seedu/address/ui/ScheduleCard.java:27]: Nope! ScheduleCardPane was inspired by cardPane from the PersonCard. `scheduleCardPanel` and `scheduleCardPane` are two different things

File: src/main/java/seedu/address/ui/ScheduleCard.java (hunk 3 above)
Comment (seanflyyy) [reply src/main/java/seedu/address/ui/ScheduleCard.java:36]: Thanks for pointing it out! Will resolve

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (seanflyyy) [reply src/main/java/seedu/address/ui/MainWindow.java:38]: Sure, will fix it

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 2 above)
Comment (seanflyyy) [reply src/main/java/seedu/address/ui/MainWindow.java:52]: I don't think so. Thanks for pointing it out

**Pull request merged**